addons/account/wizard/account_reconcile.py

```diff
@@ -56,15 +58,22 @@ def trans_rec_get(self, cr, uid, ids, context=None):
         if context is None:
             context = {}
         credit = debit = 0
-        account_id = False
+        account = False
         count = 0
         for line in account_move_line_obj.browse(cr, uid, context['active_ids'], context=context):
             if not line.reconcile_id and not line.reconcile_id.id:
                 count += 1
                 credit += line.credit
                 debit += line.debit
-                account_id = line.account_id.id
-        return {'trans_nbr': count, 'account_id': account_id, 'credit': credit, 'debit': debit, 'writeoff': debit - credit}
+                account = line.account_id
+        # If the write-off is lower than the account precision,
+        # set it to 0.
+        currency = account.company_id.currency_id
```

Comment:
potentially account is False here and we will have an AttributeError exception. 

am I missing something?